### DISCUSSION TITLE
Posssible to get test stdout from a test hook?

### DISCUSSION THREAD

**Discussion by ajeetd** [pytest-dev/pytest#10594] [Q&A]
Specifically rather than access stdout from the test function as shown here: https://docs.pytest.org/en/7.1.x/how-to/capture-stdout-stderr.html#captures I am interested in accessing stdout from pytest_runtest_teardown(item, nextitem)[[source]](https://docs.pytest.org/en/7.1.x/_modules/_pytest/hookspec.html#pytest_runtest_teardown) or a similar hook.

**Comment by arthuRHD** [comment]
You want to capture the test result or just stdout streamed while the test was executing ?
